server/website/script/fixture_generators/knob_settings/oracle/create_knob_settings.py

```diff
@@ -0,0 +1,218 @@
+        fields['default'] = 500000000  # 500M
+    if fields['name'].upper() == 'PGA_AGGREGATE_TARGET':
+        fields['tunable'] = False
+        fields['minval'] = 0
+        fields['maxval'] = 33000000000  # 33G
+        fields['default'] = 0
+    if fields['name'].lower() == 'bitmap_merge_area_size':
+        fields['tunable'] = True
+        fields['minval'] = 0
+        fields['maxval'] = 5000000000  # 3G
+        fields['default'] = 0
+    if fields['name'].lower() == 'create_bitmap_area_size':
+        fields['tunable'] = True
+        fields['minval'] = 0
+        fields['maxval'] = 5000000000  # 3G
+        fields['default'] = 0
+    if fields['name'].lower() == 'hash_area_size':
+        fields['tunable'] = True
+        fields['minval'] = 0
+        fields['maxval'] = 3000000000  # 3G
+        fields['default'] = 0
+    if fields['name'].lower() == 'sort_area_size':
+        fields['tunable'] = True
+        fields['minval'] = 0
+        fields['maxval'] = 3000000000  # 3G
+        fields['default'] = 0
+    if fields['name'].upper() == 'OPEN_CURSORS':
+        fields['tunable'] = False
+        fields['minval'] = 200
+        fields['maxval'] = 400
+        fields['default'] = 300
+    if fields['name'].upper() == 'DB_FILE_MULTIBLOCK_READ_COUNT':
+        fields['tunable'] = False
+        fields['minval'] = 64
+        fields['maxval'] = 256
+        fields['default'] = 128
+    if fields['name'].upper() == 'optimizer_index_cost_adj'.upper():
+        fields['tunable'] = False
+        fields['minval'] = 1
+        fields['maxval'] = 10000
+        fields['default'] = 100
+    if fields['name'].upper() == 'OPTIMIZER_USE_PENDING_STATISTICS':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = False
+    if fields['name'].upper() == 'OPTIMIZER_USE_INVISIBLE_INDEXES':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = False
+    if fields['name'].upper() == 'OPTIMIZER_USE_SQL_PLAN_BASELINES':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = True
+    if fields['name'].upper() == 'OPTIMIZER_CAPTURE_SQL_PLAN_BASELINES':
+        fields['tunable'] = False
+        fields['minval'] = None
+        fields['maxval'] = None
+        fields['default'] = False
+    if fields['name'].upper() == 'DISK_ASYNCH_IO':
+        fields['tunable'] = True
+        fields['vartype'] = 5
+        fields['enumvals'] = 'TRUE,FALSE'
+        fields['default'] = 'TRUE'
+
+
+def main():
+    final_metrics = []
+    with open('oracle.txt', 'r') as f:
+        num = 0
+
+        lines = f.readlines()
+        for line in lines:
+            line = line.replace("\n", "")
+            pattern = re.compile(r"^\s+|\s+$")
+            line = re.sub(pattern, '', line)
+            if len(line) <= 0:
```

Comment:
The line cannot be less than zero. Aside from that, you could just use
```suggestion
            if line:
```

Reply:
If not line: